android/src/main/java/io/rownd/android/views/RowndWebView.kt

```diff
@@ -93,11 +95,31 @@ class RowndWebView(context: Context, attrs: AttributeSet?) : WebView(context, at
     }
 }
 
-class RowndWebViewClient(webView: RowndWebView) : WebViewClient() {
+class RowndWebViewClient(webView: RowndWebView, context: Context) : WebViewClient() {
     private val webView: RowndWebView
+    private val context: Context
+    private var timeout: Boolean = true
 
     init {
         this.webView = webView
+        this.context = context
+
+        Thread {
```

Comment:
Any chance we can turn this into a coroutine instead? Might be more efficient.

Reply:
Update to a Coroutine